packages/grpc-js/src/xds-client.ts

```diff
@@ -581,16 +820,26 @@ export class XdsClient {
     }
   }
 
+  private updateRdsNames() {
+    this.adsCall?.write({
```

Comment:
I noticed that some other occurrences above uses `if (this.adsCall) { this.adsCall.write({ ...`. So perhaps those can be replaced with `this.adsCall?.write({ ...`?

Reply:
I think that replacement would be good, but now that I have the whole picture of the `XdsClient` functionality, I want to do a separate PR to refactor the class more broadly. I can make that change there.